vm/src/pyobject.rs

```diff
@@ -1196,10 +1197,12 @@ where
 }
 
 pub trait PyClassImpl: PyClassDef {
+    const TP_FLAGS: u64;
```

Comment:
You could give this a default value. so that you don't have to do it for `pystruct_sequence` or other pyclass derivatives we might have in the future